.github/workflows/ci.yml

```diff
@@ -11,7 +11,7 @@ on:
 
 jobs:
   lint:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-18.04
```

Comment:
Does `pe-parse` not build correctly on `ubuntu-latest`? Or is this just so that we actually pin to a specific version for builds? 🙂

Reply:
This was changed because I didn't know if we wanted to keep testing on Ubuntu 18.04, since the `ubuntu-latest` is changing from 18.04 to 20.04 soon™️ https://github.com/actions/virtual-environments/issues/1816

Reply:
Makes sense!